Layouts/layout-details-Phishing.json

```diff
@@ -1,347 +1,755 @@
 {
-	"typeId": "Phishing",
+	"TypeName": "Phishing",
+    "fromVersion": "5.0.0",
```

Comment:
we should add new layout files, not replacing the existing ones - since we still have users with 4.5 - we don't want to delete existing layouts

Reply:
i'm not deleting, the old one moved to be under `Layouts/layout-details-Phishing_4_5_0.json`

Reply:
😅